### ISSUE TITLE
Profile - "Go to Expensify Classic" and "Help" row is missing external link icon

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#35905]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4-37.0
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Go to staging.new.expensify.com
2. Click Profile
3. Scroll down to "Go to Expensify Classic" and "Help" row

## Expected Result:
There will be an external link icon for "Go to Expensify Classic" and "Help" row

## Actual Result:
"Go to Expensify Classic" and "Help" row is missing external link icon

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

![Bug6369367_1707228554931!Old_UI](https://github.com/Expensify/App/assets/78819774/62893dd3-42f2-42e5-8206-20040fc09024)

![Bug6369367_1707228554911!New_UI](https://github.com/Expensify/App/assets/78819774/d556445f-aa4d-443d-ae9c-f8e0326ba2f4)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
We think that this bug might be related to [#wave8-collect-admins](https://expensify.enterprise.slack.com/archives/C036QM0SLJK)
CC @zanyrenney

**Comment by MahmudjonToraqulov** [comment]
# Proposal
## Please re-state the problem that we are trying to solve in this issue.
Profile - "Go to Expensify Classic" and "Help" row is missing external link

## What is the root cause of that problem?
It's left here.

## What changes do you think we should make in order to solve the problem?
We should add the cod.

## What alternative solutions did you explore? (Optional)
N/A

**Comment by bernhardoj** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The go to expensify classic doesn't have the external link icon

### What is the root cause of that problem?
The menu doesn't have the right icon like in other parts of the app.
https://github.com/Expensify/App/blob/0eb6d02b12569c42eca494d96c0c5f15eaa0b26f/src/pages/settings/InitialSettingsPage.js#L169-L175

### What changes do you think we should make in order to solve the problem?

Add `iconRight: Expensicons.NewWindow,`

**Comment by FitseTLT** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
 "Go to Expensify Classic" and "Help" row is missing external link icon

### What is the root cause of that problem?
We are missing iconRight here 
https://github.com/Expensify/App/blob/0eb6d02b12569c42eca494d96c0c5f15eaa0b26f/src/pages/settings/InitialSettingsPage.js#L168-L175
and here
https://github.com/Expensify/App/blob/0eb6d02b12569c42eca494d96c0c5f15eaa0b26f/src/pages/settings/InitialSettingsPage.js#L198-L205
### What changes do you think we should make in order to solve the problem?

We need to pass in both places
```
                iconRight: Expensicons.NewWindow,
                shouldShowRightIcon:true

```
And here we pass
https://github.com/Expensify/App/blob/0eb6d02b12569c42eca494d96c0c5f15eaa0b26f/src/pages/settings/InitialSettingsPage.js#L263-L264
```
iconRight={item.iconRight}
shouldShowRightIcon={!!item.iconRight}
```
### What alternative solutions did you explore? (Optional)

**Comment by s77rt** [comment]
@Expensify/design This was brought up here https://github.com/Expensify/App/pull/33280#issuecomment-1876937292 and https://github.com/Expensify/App/pull/33280#issuecomment-1876942405. We didn't get a confirmation on whether this is by design or something that should be fixed.

**Comment by kosmydel** [comment]
> Rather than making this any kind of paid bug to fix, can we just add this to the account settings project? cc @kosmydel @mountiny @trjExpensify

Yeah, I think we can handle it.

**Comment by dannymcclain** [comment]
Yes please do. I think the designs not including it before was an error. The account settings mocks should be correct and show the new-window icon.

**Comment by trjExpensify** [comment]
Agreed. @kosmydel want to put it in yours? Link it here, and I'll also assign you and I to this issue to track until it's donezo.

**Comment by dannymcclain** [comment]
@mountiny This looks like it's correct on staging now, but Melvin is goin' wild 😂

**Comment by mountiny** [comment]
@dannymcclain its on staging right :D

**Comment by trjExpensify** [comment]
Hm, so what happened here? 

<img width="372" alt="image" src="https://github.com/Expensify/App/assets/16232057/02ea2d7e-7630-440a-b583-cfca09e7de22">

**Comment by trjExpensify** [comment]
Did `Go to Expensify Classic` get replaced with "Switch to Expensify Classic" and the external link icon removed?

**Comment by kosmydel** [comment]
> Did `Go to Expensify Classic` get replaced with "Switch to Expensify Classic" and the external link icon removed?

I think it was changed in [this PR](https://github.com/Expensify/App/pull/34925).

**Comment by trjExpensify** [comment]
Gotcha! @shawnborton looks like you were on that PR, not sure if you peaked it though. 

1. Are we going to leave it there as the top most row in account settings versus in the `general` section? 
2. Either way on #1, should we add back the external icon? 

CC: @roryabraham

**Comment by shawnborton** [comment]
Oh boy... I think we have some overlap here with the hybrid app project. Maybe @AndrewGable can comment, but I can see where we wanted to keep the Switch to classic towards the top so it's easy to find your way back.

That being said... maybe we don't need the external icon because I don't think we open up a new window when we switch, right? It should just feel like normal app navigation?

**Comment by dannymcclain** [comment]
>That being said... maybe we don't need the external icon because I don't think we open up a new window when we switch, right? It should just feel like normal app navigation?

Yeah if it switches you "in place" then I don't think we need the external icon. Although _I guess_ I could see an argument that you're "leaving" NewDot so... 🤷‍♂️

I don't have super strong feelings.

**Comment by trjExpensify** [comment]
Ah yeah. but on web/desktop, doesn't it open a new tab versus a redirect to Classic on the same tab. Or is that changing as well?

>Maybe @AndrewGable can comment, but I can see where we wanted to keep the Switch to classic towards the top so it's easy to find your way back.

Hm yeah, fair enough. I can see that if you're an existing user being migrated over and don't know what's going on.

**Comment by AndrewGable** [comment]
I don't have a personal preference on which icon we use, but I would like to keep the "Switch to Expensify Classic" as the first row when we are on HybridApp. I am less passionate about where it is located on non-HybridApp (on web or desktop).

**Comment by dannymcclain** [comment]
I think I'm voting no "external link" icon.

>I don't have a personal preference on which icon we use, but I would like to keep the "Switch to Expensify Classic" as the first row when we are on HybridApp. I am less passionate about where it is located on non-HybridApp (on web or desktop).

Don't we want all the apps to be the same? Or am I misunderstanding something?

**Comment by AndrewGable** [comment]
We want all apps to be the same, but these buttons will do different things on web vs HybridApp. I'm for keeping it as the first row on both.

**Comment by s77rt** [comment]
Let's close this

**Comment by mountiny** [comment]
From the comments above it seems like we can close. Please reopen if you feel otherwise
